File: cw/python/super_duper_easy.py
# https://www.codewars.com/kata/55a5bfaa756cfede78000026/train/python

def problem(a):
    try:
        a = float(a)
        return ((a * 50) + 6)
    except ValueError:
        return "Error"

a = input()
print(problem(a))

File: codewars/lua/super_duper_easy.lua
-- https://www.codewars.com/kata/55a5bfaa756cfede78000026/train/lua


function problem(a)
    if (type(a) == "string") then
        return "Error"
    end

    return a * 50 + 6;
end

print(problem(12))


return problem
